pages/api/taxas/v1/[sigla].js

```diff
@@ -0,0 +1,24 @@
+import app from '@/app';
+import NotFoundError from '@/errors/NotFoundError';
+import * as selic from 'selic';
+
+const action = async (request, response) => {
+  const sigla = request.query.sigla.toLowerCase();
+  const taxas = await selic.getRatesList();
+
+  const item = taxas.find((taxa) => taxa.name.toLowerCase() === sigla);
+
+  if (!item) {
+    response.status(404);
+    throw new NotFoundError({
+      name: 'NotFoundError',
+      message: 'Taxa ou Índice não encontrada.',
+      type: 'not_found',
+    });
+  }
+
+  response.status(200);
+  response.json({ nome: item.name, valor: item.apy });
```

Comment:
Só dúvida se não precisa dar um return aqui
```suggestion
  return response.json({ nome: item.name, valor: item.apy });
```

Reply:
Não precisa de return. Request é um callback…

Reply:
Curto bastante return aliado a response mesmo sendo callback tbm, mas não acho que é um blocker pro PR não!